database/query/eval_history.sql

```diff
@@ -219,25 +191,11 @@ SELECT s.id::uuid AS evaluation_id,
 SELECT s.evaluation_time,
        s.id,
        ere.rule_id,
-       -- entity type
-       CAST(
-	 CASE
-	 WHEN ere.repository_id IS NOT NULL THEN 1
-	 WHEN ere.pull_request_id IS NOT NULL THEN 2
-	 WHEN ere.artifact_id IS NOT NULL THEN 3
-	 END AS integer
-       ) AS entity_type,
-       -- entity id
-       CAST(
-         CASE
-         WHEN ere.repository_id IS NOT NULL THEN ere.repository_id
-         WHEN ere.pull_request_id IS NOT NULL THEN ere.pull_request_id
-         WHEN ere.artifact_id IS NOT NULL THEN ere.artifact_id
-         END AS uuid
-       ) AS entity_id,
-       ere.entity_instance_id as new_entity_id
+       ei.entity_type,
+       ere.entity_instance_id as entity_id
   FROM evaluation_statuses s
        JOIN evaluation_rule_entities ere ON s.rule_entity_id = ere.id
+       LEFT JOIN entity_instances AS ei ON ei.id = ere.entity_instance_id
```

Comment:
**nit:** IIRC there is an `ON DELETE CASCADE` clause in the foreign key of from `evaluation_rule_entities`, so this should be a plain `JOIN`.

Is there a particular reason why this is a `LEFT JOIN` instead? Was data migrated to the new table?